va/Android.mk

```diff
@@ -25,8 +25,8 @@
 
 LOCAL_PATH:= $(call my-dir)
 
-LIBVA_DRIVERS_PATH_32 := /vendor/lib/dri
-LIBVA_DRIVERS_PATH_64 := /vendor/lib64/dri
+LIBVA_DRIVERS_PATH_32 := /system/lib/dri
```

Comment:
libva should be put at vendor partition since system partition is for android AOSP and can be updated seperately with vendor partition.